static/js/event/page.tsx

```diff
@@ -20,6 +20,43 @@
 
 import React from "react";
 
+import { Property } from "./types";
+
 export function Page(): JSX.Element {
   return <div></div>;
 }
+
+interface EventPagePropsType {
+  dcid: string;
+  name: string;
+  properties: Array<Property>;
+}
+
+export function EventPage(props: EventPagePropsType): JSX.Element {
+  return (
+    <div>
+      <h1>{props.name}</h1>
+      <h3>dcid: {props.dcid}</h3>
+      <br />
+      <h3>Properties:</h3>
+      {props.properties.map((property) => {
+        return (
+          <div key={property.name}>
```

Comment:
nit: name works, but I would prefer to use dcid in the key because dcid is guaranteed to be unique

Reply:
property.name actually referred to dcid. Updated the naming to make this clear.